src/Movie/Top-Bar/Top-Bar.jsx

```diff
@@ -0,0 +1,28 @@
+import React from "react";
+import './Top-Bar.css'
+import { useRecoilState } from 'recoil';
+import { userAtom } from '../../Login/atoms';
+function TopBar() {
+    const [user , setUser] = useRecoilState(userAtom)
+    const handleClick = (event) => {
+        event.preventDefault();
+        setUser(null);
+        localStorage.removeItem('auth');
+
+    }
+    return (
+        <div className="topbar">
+            <h1>
+                Movies
+            </h1>
+            <div className="right-box">
+                <li className="top-bar-list">
+                    <a onClick={handleClick} href="" >Logout</a>
```

Comment:
[low]
use react router for navigating and not use anchor tag if not want to open external link